public/develop/intellij.md

```diff
@@ -0,0 +1,85 @@
+# IntelliJ
+
+A rough guide on how to develop molgenis in IntelliJ IDEA.
+This was created using IntelliJ IDEA 2016.1 Ultimate, courteously provided to us by
+JetBrains on an open source license.
+
+## Middleware
+Before you start, make sure the following middleware is installed on your system.
+* MySQL server.
+    * Create a database called molgenis with a user molgenis that has full permissions on the database.
+* Java 8 JDK (latest version available from Oracle)
+* Maven >= 3.1.0 (The built-in maven version of IntelliJ is too old to compile molgenis.)
+* Apache Tomcat 7 or 8
+* MySQL JDBC driver. Put it in the lib/ext dir of your Tomcat installation.
+
+## Molgenis settings
+You'll need a molgenis-server.properties file. In your home dir, create a folder
+`.molgenis`. In it create a folder called `omx`.
+Put a file in there called `molgenis-server.properties` and fill it with the
```

Comment:
The latter I think, or else we need to update everything the moment this changes.